frontend/components/TableContainer/TableContainer.tsx

```diff
@@ -100,6 +101,7 @@ const TableContainer = ({
   isLoading,
   manualSortBy = false,
   defaultSearchQuery = "",
+  defaultPageIndex = 0,
```

Comment:
You could make this line `defaultPageIndex = DEFAULT_PAGE_INDEX` if we want to make use of the constant defined above.